src/DynamoCore/Utilities/Configurations.cs

```diff
@@ -22,14 +22,12 @@ public class Configurations
         public static string DynamoWikiLink = "https://github.com/DynamoDS/Dynamo/wiki";
         public static string DynamoBimForum = "http://dynamobim.org/forums/forum/dyn/";
         public static string DynamoTeamEmail = "mailto:team@dynamobim.org";
-        public static string DynamoAdvancedTutorials = "http://dynamobim.org/learn/#7570";
+        public static string DynamoAdvancedTutorials = "http://dynamobim.org/learn/#124";
```

Comment:
Oops, sorry I think you're absolutely right, good catch! It should have been `7570`! Updated.